src/Plugins/Issue.php

```diff
@@ -0,0 +1,169 @@
+<?php declare(strict_types=1);
+
+// TODO - Scrape for GH profiles and ping detect
+  
+namespace Room11\Jeeves\Plugins;
+
+use Amp\Success;
+use Amp\Artax\HttpClient;
+use Amp\Artax\Request as HttpRequest;
+use Room11\Jeeves\External\GithubIssue\Credentials;
+use Room11\Jeeves\Chat\Client\ChatClient;
+use Room11\Jeeves\Chat\Message\Command;
+use Room11\Jeeves\System\PluginCommandEndpoint;
+use Room11\Jeeves\Chat\Client\MessageResolver;
+use Room11\Jeeves\Storage\KeyValue as KeyValueStore;
+
+class Issue extends BasePlugin
+{
+    const USAGE = 'Usage: `!!issue [<title> - <body>]`';
+    const ISSUE_URL = 'https://api.github.com/repos/Room-11/Jeeves/issues';
+    const CHAT_URL_EXP = '~^http://chat\.stackoverflow\.com/transcript/message/(\d+)(#\d+)?$~';
+    const PING_EXP = '/@([^\s]+)(?=$|\s)/';
+    const REQUIRED_AUTH = ['username', 'password'];
+
+    private $chatClient;
+    private $httpClient;
+    private $credentials;
+    private $response;
+
+    public function __construct(
+        ChatClient $chatClient, 
+        MessageResolver $messageResolver,
+        HttpClient $httpClient,
+        Credentials $credentials
+    ) {
+        $this->credentials = $credentials;
+        $this->httpClient = $httpClient;
+        $this->messageResolver = $messageResolver;
+        $this->chatClient = $chatClient;
+    }
+
+    public function issue(Command $command)
+    {
+        if (!$this->credentialsPresent()) {
+            return $this->chatClient->postReply(
+                $command, 'I\'m not configured to use GitHub :('
+            );
+        }
+
+        $content = explode('-', $command->getText());
+
+        if (empty($content[0]) || count($content) > 2) {
+            return $this->chatClient->postReply($command, self::USAGE);
+        }
+        
+        $title = yield from $this->getTextFromCommand($content[0], $command);
+        $body = null;
+
+        if (isset($content[1])) {
+            $body = yield from $this->getTextFromCommand($content[1], $command);
+        }
+
+        yield from $this->createIssue($title, $body, $command->getId());
+
+        return $this->chatClient->postReply($command, $this->response);
+    }
+
+    private function createIssue(string $title, $body = '', int $id)
+    { // TODO Handle different status returns
```

Comment:
As discussed, we're happy with just not working / working for now. Though I could add some logging to make it easier to debug